tools/generator-bot-template-generator/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "generator-bot-template-generator",
+  "version": "0.0.1",
+  "description": "Generate a bot template generator for use with Microsoft Bot Framework",
+  "author": "Microsoft",
+  "files": [
+    "app"
+  ],
+  "keywords": [
+    "bot-template-generator",
+    "microsoft bot template"
```

Comment:
Should these not be separated out into "microsoft" "bot" "template"?

Reply:
Andrew has a task next week to go through and consolidate all of the package naming, keywords, etc.

Reply:
Handled